.k8/helm3/ssv-node-cluster-b/node-b-1/templates/node-b-1-deployment.yaml

```diff
@@ -2,10 +2,10 @@
 apiVersion: v1
```

Comment:
I would really rather that we not have so many files, e.g for each cluster and node. if we could have a general file for a cluster and node and run it multiple times with different arguments (which name will be one of them). it will be much cleaner